File: Codes in C++/Graph/DFS.cpp
class Solution {
  public:
    // Function to return a list containing the DFS traversal of the graph.
    void dfs(int V,vector<int> adj[],vector<int> &vis,vector<int> &res){
            vis[V]=1;
            res.push_back(V);
        for (auto it:adj[V])
            if(!vis[it])
            dfs(it,adj,vis,res);
        
            
    }
    vector<int> dfsOfGraph(int V, vector<int> adj[]) {
        vector<int> res;
        vector<int> vis(V,0);
        dfs(0,adj,vis,res);
        return res;
    }
};

File: Codes in Python3/Graph/DFS.py
class Solution:
    def dfs(self,V,adj,vis,res):
        vis[V]=True
        res.append(V)
        for i in adj[V]:
            if vis[i] is False:
                self.dfs(i,adj,vis,res)
                
    #Function to return a list containing the DFS traversal of the graph.
    def dfsOfGraph(self, V, adj):
        res=[]
        vis=[False for i in range(V)]
        self.dfs(0,adj,vis,res)
        return res